HACKING.md

```diff
@@ -37,14 +32,8 @@ tests. Look up their names in the Check documentation:
 
 ##Write unit tests for the tools
 
-If you don't feel too comfortable writing C code, you could help in
-writing unit tests for tool/dime and tool/signet. These unit tests
-should be written in bash or some other scripting language. We don't
-have a test framework for that, so you can choose yourself.
+If you don't feel too comfortable writing C code, you could help in writing unit tests for tool/dime and tool/signet. Write these unit tests in bash or some other scripting language. We don't have a test framework for that so that you can choose yourself.
 
 ##Fix bugs
 
-If you know how to fix bugs in C, I can give you an account on
-https://scan.coverity.com/, a static code analyzer that has found quite
-a few bugs in the code, which need to be fixed. I already fixed some of
-the simple ones, and the remaining ones get more and more trickier.
+If you know how to fix bugs in C, I can give you an account on https://scan.coverity.com/, a static code analyzer that has found quite a few bugs in the code, which need to be fixed. I already fixed some of the simple ones, and the remaining ones get more and trickier.
```

Comment:
I think `the remaining ones get more and tricky.` would be the proper way to say this.